packages/common/src/search/utils.ts

```diff
@@ -79,6 +80,59 @@ export function expandApi(api: NamedApis | IApiDefinition): IApiDefinition {
   }
 }
 
+/**
+ * @private
+ * Determines Which API should be hit for the given search parameters.
+ * Hierarchy:
+ * - Target options.api if available
+ * - Target the OGC API current parameters allow
+ * - Target the Portal API based off options.requestOptions.portal
+ * @param targetEntity target entity of the query
+ * @param options search options
+ * @returns an API Definition object describing what should be targeted
+ */
+export function getApi(
```

Comment:
can we move this and `shouldUseOgcApi` to an internal file? We should only add stuff to this file if we intend for it to be used outside of this package. If needed we can move the expandApi logic to the internal file (i.e. leave `expandApi` here and likely deprecate it, and have it call a private/internal `_expandApi()`).

Reply:
Good catch! Let me see what I can do

Reply:
done